hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableMetaClient.java

```diff
@@ -623,6 +624,8 @@ private static void initTableMetaClient(StorageConfiguration<?> storageConf, Str
     }
 
     initializeBootstrapDirsIfNotExists(basePath, storage);
+    // When the table is initialized, set the initial version to be the current version.
+    props.put(INITIAL_VERSION.key(), String.valueOf(HoodieTableVersion.current().versionCode()));
```

Comment:
Are there no other callers of this method apart from `initTable`? It's important that the INITIAL_VERSION is not changed later due to offline/manual action right.

Reply:
Good point. Found one place in RepairsCommand and added the fix.